VEHICLE/AttachVehicleToCargobob.md

```diff
@@ -5,18 +5,47 @@ ns: VEHICLE
 
 ```c
 // 0x4127F1D84E347769 0x607DC9D5
-void ATTACH_VEHICLE_TO_CARGOBOB(Vehicle vehicle, Vehicle cargobob, int p2, float x, float y, float z);
-```
-
-```
-p2 is unknown and is always -1 in the script natives.
+void ATTACH_VEHICLE_TO_CARGOBOB(Vehicle cargobob, Vehicle vehicle, int vehicleBoneIndex, float x, float y, float z);
 ```
 
 ## Parameters
-* **vehicle**: The vehicle which should be attached
 * **cargobob**: The cargobob
-* **p2**:
-* **x**:
-* **y**:
-* **z**:
+* **vehicle**: The vehicle which should be attached
+* **vehicleBoneIndex**: A Vehicle bone the hook/magnet should attach to (Can also be false) https://docs.fivem.net/natives/?_0xFB71170B7E76ACBA
```

Comment:
Shouldn't this be `A Vehicle bone the hook/magnet should attach to or -1 for none/default`

Also using relative paths with referring to natives is preferred, i.e. `[GET_ENTITY_BONE_INDEX_BY_NAME](#_0xFB71170B7E76ACBA)`

Reply:
Updated all, Thanks!